flink-end-to-end-tests/test-scripts/test_kubernetes_materialized_table.sh

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
some documentation describing this would be useful. 
This test script appears to alter the material table, i.e. change its shape - presumably with a compatible schema change.
 I was expecting the refresh to be a refresh of the data not the shape of the table - did I misunderstand.

Reply:
This e2e test is mainly to test submitting the refresh task to k8s, so there is no complicated alter operation.

Reply:
I added a simple comment to describe the script. I think the script itself is relatively simple and should not require too much explanation of its detailed process. I have seen other e2e scripts that are similar.